python-sdk/tests/benchmark/run.py

```diff
@@ -75,28 +76,44 @@ def wrapper(*args, **kwargs):
         if sys.platform == "linux":
             io_counters_before = process.io_counters()._asdict()
 
-        # run decorated function
-        dag = func(*args, **kwargs)
-
-        # metrics after
-        memory_full_info_after = process.memory_full_info()._asdict()
-        cpu_time_after = process.cpu_times()._asdict()
-        disk_usage_after = get_disk_usage()
-        if sys.platform == "linux":
-            io_counters_after = process.io_counters()._asdict()
-
-        profile = {
-            "duration": get_load_task_duration(dag=dag),
-            "memory_full_info": subtract(
-                memory_full_info_after, memory_full_info_before
-            ),
-            "cpu_time": subtract(cpu_time_after, cpu_time_before),
-            "disk_usage": disk_usage_after - disk_usage_before,
-        }
-        if sys.platform == "linux":
-            profile["io_counters"] = (subtract(io_counters_after, io_counters_before),)
+        skip = kwargs.pop("skip", False)
```

Comment:
Is this `skip` database specific?

Reply:
Nope, it's specific to database and datasets combination. Specified in benchmarking config.